indico/modules/events/editing/client/js/components/Timeline/TimelineItem.jsx

```diff
@@ -58,13 +61,20 @@ export default function TimelineItem({block}) {
                 </time>
               </div>
               {!isLastBlock && (
-                <a className="block-info-link i-link" onClick={() => setVisible(!visible)}>
-                  {visible ? (
-                    <Translate>Hide old revision</Translate>
-                  ) : (
-                    <Translate>Show old revision</Translate>
+                <>
+                  <a className="block-info-link i-link" onClick={() => setVisible(!visible)}>
+                    {visible ? (
+                      <Translate>Hide old revision</Translate>
+                    ) : (
+                      <Translate>Show old revision</Translate>
+                    )}
+                  </a>
+                  {!visible && block.finalState && (
```

Comment:
Isn't it enough to just check for `block.finalState`?

Reply:
Ah, nvm :P I know what the check is for now ;)

Reply:
feels a bit strange to me that it disappears when expanding